tests/contract-call.sh

```diff
@@ -4,36 +4,33 @@ set -eu -o pipefail
 
 # Examples:
 # contract-call.sh test_empty
-# contract-call.sh test_bool "true" "false"
+# contract-call.sh test_bool "[true, false]"
 
 CONTRACT_SRC="contracts/Test.aes"
 ADDRESS_SRC="build/${CONTRACT_SRC//.aes/.addr}"
 WALLET_PATH="build/wallet"
 WALLET_PASS="123"
-# init() calldata
-CALLDATA="cb_KxFE1kQfP4oEp9E="
```

Comment:
why did you removed the calldata param ? What if the contract needs some arguments for init?

Reply:
I thought it was extracted to explain what is it. Since, cli doesn't require arguments and it not acepting encoded arguments anymore, but I can revert it 🙂